presto-memsql/src/test/java/io/prestosql/plugin/memsql/TestMemSqlTypeMapping.java

```diff
@@ -95,7 +106,8 @@ private static DataTypeTest singlePrecisionFloatingPointTests(DataType<Float> fl
         // we are not testing Nan/-Infinity/+Infinity as those are not supported by MemSQL
         return DataTypeTest.create()
                 .addRoundTrip(floatType, 3.14f)
-                // .addRoundTrip(floatType, 3.1415927f) // Overeagerly rounded by MemSQL to 3.14159 TODO
+                // TODO Overeagerly rounded by MemSQL to 3.14159
```

Comment:
This seems to be due to MemSQL's FLOAT type having low precision rather than rounding errors.